content/en/docs/concepts/security/index.md

```diff
@@ -750,34 +754,54 @@ spec:
         notRequestPrincipals: ["*"]
 {{< /text >}}
 
-#### Allow-all and default deny-all authorization policies
+#### Default allow and deny policy
```

Comment:
Question: I find it a bit hard to associate the sub section title with the content.

The text seems describe how to get started for initial config. But the title indicates about what's the default behavior?

Reply:
That's a good question, my intention is to explain the common allow-nothing, deny-all, etc.

Updated the title to: `allow-nothing`, `deny-all` and `allow-all` policy